src/main/java/Duke.java

```diff
@@ -1,10 +1,166 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
+    static final String lineDivider = "\t__________________________________________________________________________\n";
+
+    static final String dukeKeywords = "\t\t Use 'todo', 'deadline' , 'event' to enter tasks!\n";
+    public static final String LOGO = " ____        _        \n"
+                                    + "|  _ \\ _   _| | _____ \n"
+                                    + "| | | | | | | |/ / _ \\\n"
+                                    + "| |_| | |_| |   <  __/\n"
+                                    + "|____/ \\__,_|_|\\_\\___|\n";
+
+    static final String dukeGreeting =
```

Comment:
This part is good as many tends to forget the point on keeping your single line of code short (lesser than 120). This also makes it very readable.